api/proto/config/devmodel.proto

```diff
@@ -56,3 +86,38 @@ message SystemAdapter {
 	// and used for app instances
 	string logicalName = 6;
 }
+
+enum PhyIoType {
+        PhyIoNoop = 0;
+        PhyIoEth = 1;
+        PhyIoUSB = 2;
+        PhyIoCOM = 3;
+        PhyIoOther = 255;
```

Comment:
Should we split out audio while we are at it? Or we're fine to have everything else under other?
We have Hdmi in the old/current definitions.

Reply:
Good point. I will add those 2 as well.